packages/@uppy/companion/src/server/provider/index.js

```diff
@@ -132,7 +133,13 @@ module.exports.addProviderOptions = (options, grantConfig) => {
     if (grantConfig[authProvider]) {
       // explicitly add providerOptions so users don't override other providerOptions.
       grantConfig[authProvider].key = providerOptions[authProvider].key
-      grantConfig[authProvider].secret = providerOptions[authProvider].secret
+
+      // only read and override a secret from file if such a file is not undefined
+      if (typeof providerOptions[authProvider].secretFile === 'string') {
```

Comment:
I don't think there's a need for this check here. There's no such property as `secretFile`. Also, at this point, the value of `.secret` is only available in memory (and was probably loaded with [this](https://github.com/transloadit/uppy/blob/a8e17bee5fc6a93fb2298cfca6ee45c9cc0cb7bc/packages/%40uppy/companion/src/standalone/helper.js#L35)), so there's no need to read from file anymore.

Reply:
True! That is a leftover from before switching to reading in `getConfigFromEnv`.